x-pack/plugins/infra/public/pages/logs/log_entry_categories/page_setup_content.tsx

```diff
@@ -4,98 +4,51 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { EuiSpacer, EuiSteps, EuiText } from '@elastic/eui';
+import React from 'react';
 import { FormattedMessage } from '@kbn/i18n/react';
-import React, { useMemo } from 'react';
+import { EuiText, EuiButton, EuiSpacer } from '@elastic/eui';
 
-import { BetaBadge } from '../../../components/beta_badge';
 import {
-  createInitialConfigurationStep,
-  createProcessStep,
   LogAnalysisSetupPage,
   LogAnalysisSetupPageContent,
   LogAnalysisSetupPageHeader,
 } from '../../../components/logging/log_analysis_setup';
 import { useTrackPageview } from '../../../../../observability/public';
-import { useLogEntryCategoriesSetup } from './use_log_entry_categories_setup';
 
-export const LogEntryCategoriesSetupContent: React.FunctionComponent = () => {
+interface LogEntryCategoriesSetupContentProps {
+  onOpenSetup: () => void;
+}
+
+export const LogEntryCategoriesSetupContent: React.FunctionComponent<LogEntryCategoriesSetupContentProps> = ({
+  onOpenSetup,
+}) => {
   useTrackPageview({ app: 'infra_logs', path: 'log_entry_categories_setup' });
   useTrackPageview({ app: 'infra_logs', path: 'log_entry_categories_setup', delay: 15000 });
 
-  const {
-    cleanUpAndSetUp,
-    endTime,
-    isValidating,
-    lastSetupErrorMessages,
-    setEndTime,
-    setStartTime,
-    setValidatedIndices,
-    setUp,
-    setupStatus,
-    startTime,
-    validatedIndices,
-    validationErrors,
-    viewResults,
-  } = useLogEntryCategoriesSetup();
-
-  const steps = useMemo(
-    () => [
-      createInitialConfigurationStep({
-        setStartTime,
-        setEndTime,
-        startTime,
-        endTime,
-        isValidating,
-        validatedIndices,
-        setupStatus,
-        setValidatedIndices,
-        validationErrors,
-      }),
-      createProcessStep({
-        cleanUpAndSetUp,
-        errorMessages: lastSetupErrorMessages,
-        isConfigurationValid: validationErrors.length <= 0 && !isValidating,
-        setUp,
-        setupStatus,
-        viewResults,
-      }),
-    ],
-    [
-      cleanUpAndSetUp,
-      endTime,
-      isValidating,
-      lastSetupErrorMessages,
-      setEndTime,
-      setStartTime,
-      setUp,
-      setValidatedIndices,
-      setupStatus,
-      startTime,
-      validatedIndices,
-      validationErrors,
-      viewResults,
-    ]
-  );
-
   return (
-    <LogAnalysisSetupPage data-test-subj="logEntryCategoriesSetupPage">
+    <LogAnalysisSetupPage>
       <LogAnalysisSetupPageHeader>
         <FormattedMessage
           id="xpack.infra.logs.logEntryCategories.setupTitle"
-          defaultMessage="Enable Machine Learning analysis"
-        />{' '}
-        <BetaBadge />
+          defaultMessage="Setup log category analysis"
```

Comment:
"Setup" is a noun. Since this is an action, we should use the verb "set up" here.
```suggestion
          defaultMessage="Set up log category analysis"
```